src/components/crossDevice/MobileLink/index.js

```diff
@@ -0,0 +1,56 @@
+import { h, Component} from 'preact'
+import theme from '../../Theme/style.css'
+import style from './style.css'
+import classNames from 'classnames'
+
+class MobileLink extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { copySuccess: false }
+  }
+
+  copyToClipboard = (e) => {
+    this.textArea.select()
+    document.execCommand('copy')
+    e.target.focus()
+    this.setState({ copySuccess: true})
+  };
+
+  render() {
+    // TODO this URL should point to where we host the mobile flow
+    const mobileUrl = `${document.location.origin}/?roomId=${this.props.roomId}&mobileFlow=true`
+    const buttonCopy = this.state.copySuccess ? 'Copied' : 'Copy link'
+    return (
+      <div>
+        <div className={style.container}>
+          <h1 className={`${theme.title} ${style.title}`}>Continue verification on your mobile</h1>
+          <div>Copy and send the below link to your mobile</div>
+
+          <div className={style.linkSection}>
+            <div className={style.linkTitle}>Secure link</div>
+            <div className={classNames(style.actionContainer, {[style.copySuccess]: this.state.copySuccess})}>
+              <textarea ref={(textarea) => this.textArea = textarea} value={mobileUrl} />
```

Comment:
The link is wrapping out of sight when it is too long. In production the link should be shorter, but we should make sure it never disappears from view.

Reply:
Yes, you're right. I was hoping for a shorter link that's why I didn't make the textarea expandable but I will make sure that link is always fully visible